assets/src/dashboard/app/views/storyAnimTool/timeline/index.js

```diff
@@ -59,6 +97,30 @@ function Timeline({
   onAnimationDelete,
   onToggleTargetSelect,
 }) {
+  const [formFields, setFormFields] = useState({});
+
+  const animationTypes = useMemo(() => Object.values(ANIMATION_TYPES), []);
```

Comment:
nit: can just move this outside the react component since it doesn't need any data within the closure and doesn't need to cause updates